src/cake.ts

```diff
@@ -1,10 +1,13 @@
 import { exec } from '@actions/exec';
 import { which } from '@actions/io';
-import { CakeToolSettings } from './cakeToolSettings';
+
 import { CakeParameter } from './cakeParameter';
```

Comment:
**nitpick:** the order of the `import` statements is deliberate. Please, revert this change.

Reply:
Done with another commit